.netlify/build.sh

```diff
@@ -35,7 +35,7 @@ towncrier build --yes --version Unreleased --date TBC
 MDBOOK_VERSION=$(cargo search mdbook --limit 1 | head -1 | tr -s ' ' | cut -d ' ' -f 3 | tr -d '"')
 INSTALLED_MDBOOK_VERSION=$(mdbook --version || echo "none")
 if [ "${INSTALLED_MDBOOK_VERSION}" != "mdbook v${MDBOOK_VERSION}" ]; then
```

Comment:
Sorry for being late (missing the notification somehow), but for recent Cargo versions, isn't `cargo install` a no-op if the relevant version is already installed? (At least as long as Cargo's metadata like `.crates2.json` in `$CARGO_HOME` is fully present.)

Reply:
Ah, quite possibly. I kinda just bodged together the first thing which worked for this build script 😂